test/test.spec.ts

```diff
@@ -742,6 +743,41 @@ describe('Protocol', function () {
               });
           });
       });
+      
+      it('POST /dashboard/load returns 400 if the request is invalid', function (done) {
```

Comment:
Do you think it would make sense to group these dashboard tests similar to how you did it for the tests below?

It seems the load is the only place where the report and dashboard tests overlap and they would eventually become a separate describe block anyways.  

I could see either keeping them in pairs has advantage of keeping tests that are written with same setup nearby vs test that are related to same context logically grouped.  If it's easy change go ahead otherwise keep it as is.